policyengine_us/parameters/gov/states/va/dss/tanf/grant_standard/group2/addition.yaml

```diff
@@ -0,0 +1,12 @@
+description: VA provides extra assistance for each person above 10 in a family living in group II localities of this amount.
+
+values:
+  2020-07-01: 81
```

Comment:
I think we need values that start from 2021.

Reply:
Let's reflect the as-of date based on the reference